torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -788,7 +855,9 @@ struct CudaGraphFuser {
 
     // TODO: failure in buildShapeExpressions should not break fusion execution,
     // we can add a try/catch here to bailout from removeOutputsUsedOnlyInSize.
+    GRAPH_DUMP("before build shape expression: ", graph_);
```

Comment:
does this unconditionally dump the graph?

Reply:
Nope, this is only enabled via environment variable: https://github.com/pytorch/pytorch/blob/cbc34db39e6b4d2a9c685ba25416ed7e6df05890/torch/csrc/jit/jit_log.h#L6-L37